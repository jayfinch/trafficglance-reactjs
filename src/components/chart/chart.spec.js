import React from 'react'
import Chart from './chart'
import renderer from 'react-test-renderer'

describe('<Chart />', () => {
  it('Should render duration segments', () => {
    let tree = renderer.create(
      <Chart
        data={{
          durationSeriousCongestion: 1,
          durationModerateCongestion: 2,
          durationLowCongestion: 3,
          durationNoCongestion: 4
        }}
      />
    ).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
