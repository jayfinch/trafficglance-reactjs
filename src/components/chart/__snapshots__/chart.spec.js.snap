exports[`<Chart /> Should render duration segments 1`] = `
<div
  className={undefined}
  data={
    Object {
      "className": "img-responsive",
      "data": Array [
        Object {
          "color": "#e01e1b",
          "value": 1,
        },
        Object {
          "color": "#fc8d0d",
          "value": 2,
        },
        Object {
          "color": "#ede725",
          "value": 3,
        },
        Object {
          "color": "#45c43f",
          "value": 4,
        },
      ],
      "height": "90",
      "options": Object {
        "animateRotate": false,
        "animateScale": true,
        "animation": true,
        "animationSteps": 40,
        "percentageInnerCutout": 70,
        "segmentShowStroke": false,
        "showTooltips": false,
      },
      "width": "90",
    }
  }
  height={undefined}
  options={Object {}}
  width={
    Object {
      "enqueueCallback": [Function],
      "enqueueCallbackInternal": [Function],
      "enqueueElementInternal": [Function],
      "enqueueForceUpdate": [Function],
      "enqueueReplaceState": [Function],
      "enqueueSetState": [Function],
      "isMounted": [Function],
      "validateCallback": [Function],
    }
  } />
`;
